content/en/examples/concepts/policy/limit-range/problematic-limit-range.yaml

```diff
@@ -12,3 +12,4 @@ spec:
       cpu: "1"
     min:
       cpu: 100m
+    type: Container
```

Comment:
I think, there's already a PR for this #38706

Reply:
Yup. This fix was triggered by that PR. I didn't realize that there are example manifests not covered by the test case.
I'm happy to rebase this on that one.

Reply:
That would work, thank you :) I think the other PR just requires approval from someone since I've already given it an LGTM.

Reply:
Okay. #38706 is in now.